<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1020</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Formirovanie</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QTableWidget" name="list">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <attribute name="horizontalHeaderMinimumSectionSize">
           <number>18</number>
          </attribute>
          <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderDefaultSectionSize">
           <number>18</number>
          </attribute>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="toRightBtn">
            <property name="minimumSize">
             <size>
              <width>24</width>
              <height>40</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>24</width>
              <height>40</height>
             </size>
            </property>
            <property name="text">
             <string>&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="toLeftBtn">
            <property name="minimumSize">
             <size>
              <width>24</width>
              <height>40</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>24</width>
              <height>40</height>
             </size>
            </property>
            <property name="text">
             <string>&lt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="allRightBtn">
            <property name="minimumSize">
             <size>
              <width>24</width>
              <height>40</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>24</width>
              <height>40</height>
             </size>
            </property>
            <property name="text">
             <string>&gt;&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="allLeftBtn">
            <property name="minimumSize">
             <size>
              <width>24</width>
              <height>40</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>24</width>
              <height>40</height>
             </size>
            </property>
            <property name="text">
             <string>&lt;&lt;</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableWidget" name="buffer">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <attribute name="horizontalHeaderMinimumSectionSize">
           <number>18</number>
          </attribute>
          <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderDefaultSectionSize">
           <number>18</number>
          </attribute>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QGroupBox" name="search_group">
          <property name="minimumSize">
           <size>
            <width>300</width>
            <height>200</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>180</width>
            <height>200</height>
           </size>
          </property>
          <property name="title">
           <string>Поиск</string>
          </property>
          <widget class="QPushButton" name="search_submit">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>170</y>
             <width>281</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>Поиск</string>
           </property>
          </widget>
          <widget class="QComboBox" name="search_district">
           <property name="geometry">
            <rect>
             <x>60</x>
             <y>50</y>
             <width>161</width>
             <height>22</height>
            </rect>
           </property>
          </widget>
          <widget class="QDateEdit" name="search_arrive">
           <property name="geometry">
            <rect>
             <x>60</x>
             <y>80</y>
             <width>81</width>
             <height>22</height>
            </rect>
           </property>
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QDateEdit" name="search_despature">
           <property name="geometry">
            <rect>
             <x>140</x>
             <y>80</y>
             <width>81</width>
             <height>22</height>
            </rect>
           </property>
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QLineEdit" name="search_secName">
           <property name="geometry">
            <rect>
             <x>60</x>
             <y>20</y>
             <width>161</width>
             <height>20</height>
            </rect>
           </property>
          </widget>
          <widget class="QLineEdit" name="search_token">
           <property name="geometry">
            <rect>
             <x>60</x>
             <y>110</y>
             <width>161</width>
             <height>20</height>
            </rect>
           </property>
          </widget>
          <widget class="QLineEdit" name="search_vbnum">
           <property name="geometry">
            <rect>
             <x>60</x>
             <y>140</y>
             <width>161</width>
             <height>20</height>
            </rect>
           </property>
          </widget>
          <widget class="QLabel" name="label_2">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>20</y>
             <width>46</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Фамилия</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_3">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>50</y>
             <width>46</width>
             <height>21</height>
            </rect>
           </property>
           <property name="text">
            <string>Район</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_4">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>80</y>
             <width>51</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Прибыл/</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_5">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>110</y>
             <width>46</width>
             <height>21</height>
            </rect>
           </property>
           <property name="text">
            <string>Жетон</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_6">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>140</y>
             <width>21</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>В/Б</string>
           </property>
          </widget>
          <widget class="QRadioButton" name="search_radioALL">
           <property name="geometry">
            <rect>
             <x>230</x>
             <y>20</y>
             <width>82</width>
             <height>17</height>
            </rect>
           </property>
           <property name="text">
            <string>Везде</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroup</string>
           </attribute>
          </widget>
          <widget class="QRadioButton" name="search_radioRez">
           <property name="geometry">
            <rect>
             <x>230</x>
             <y>40</y>
             <width>82</width>
             <height>17</height>
            </rect>
           </property>
           <property name="text">
            <string>Резерв</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroup</string>
           </attribute>
          </widget>
          <widget class="QLabel" name="label_7">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>90</y>
             <width>31</width>
             <height>20</height>
            </rect>
           </property>
           <property name="text">
            <string>убыл</string>
           </property>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>100</height>
           </size>
          </property>
          <property name="title">
           <string>Команды</string>
          </property>
          <widget class="QTableWidget" name="commandList">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>50</y>
             <width>671</width>
             <height>141</height>
            </rect>
           </property>
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="alternatingRowColors">
            <bool>false</bool>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::SingleSelection</enum>
           </property>
           <property name="selectionBehavior">
            <enum>QAbstractItemView::SelectRows</enum>
           </property>
           <property name="sortingEnabled">
            <bool>true</bool>
           </property>
           <attribute name="horizontalHeaderMinimumSectionSize">
            <number>18</number>
           </attribute>
           <attribute name="horizontalHeaderStretchLastSection">
            <bool>true</bool>
           </attribute>
           <attribute name="verticalHeaderDefaultSectionSize">
            <number>18</number>
           </attribute>
           <attribute name="verticalHeaderMinimumSectionSize">
            <number>18</number>
           </attribute>
           <attribute name="verticalHeaderStretchLastSection">
            <bool>false</bool>
           </attribute>
          </widget>
          <widget class="QComboBox" name="orders">
           <property name="geometry">
            <rect>
             <x>560</x>
             <y>20</y>
             <width>121</width>
             <height>22</height>
            </rect>
           </property>
          </widget>
          <widget class="QLabel" name="label">
           <property name="geometry">
            <rect>
             <x>520</x>
             <y>20</y>
             <width>41</width>
             <height>20</height>
            </rect>
           </property>
           <property name="text">
            <string>Приказ</string>
           </property>
          </widget>
          <widget class="QPushButton" name="bandCommand">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>20</y>
             <width>101</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>Сформировать</string>
           </property>
          </widget>
          <widget class="QPushButton" name="disbandCommand">
           <property name="geometry">
            <rect>
             <x>120</x>
             <y>20</y>
             <width>111</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>Расформировать</string>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1020</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Призывники</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Команды</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>Отчеты</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_5">
    <property name="title">
     <string>Настройка</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_6">
    <property name="title">
     <string>Справка</string>
    </property>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
   <addaction name="menu_5"/>
   <addaction name="menu_6"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="import_act"/>
   <addaction name="add_body_act"/>
   <addaction name="export_excel_act"/>
   <addaction name="sleeping_list_act"/>
   <addaction name="act_command"/>
  </widget>
  <action name="import_act">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/img/import.png</normaloff>:/img/import.png</iconset>
   </property>
   <property name="text">
    <string>Импорт</string>
   </property>
   <property name="toolTip">
    <string>Импорт данных о призывниках</string>
   </property>
  </action>
  <action name="add_body_act">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/img/add_body.ico</normaloff>:/img/add_body.ico</iconset>
   </property>
   <property name="text">
    <string>Добавить призывника</string>
   </property>
   <property name="toolTip">
    <string>Добавить призывника в реестр</string>
   </property>
   <property name="shortcut">
    <string>F3</string>
   </property>
  </action>
  <action name="export_excel_act">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/img/export_excel.png</normaloff>:/img/export_excel.png</iconset>
   </property>
   <property name="text">
    <string>экспорт в excel</string>
   </property>
   <property name="toolTip">
    <string>Экспорт в excel</string>
   </property>
  </action>
  <action name="sleeping_list_act">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/img/sleep.png</normaloff>:/img/sleep.png</iconset>
   </property>
   <property name="text">
    <string>Список ночующих</string>
   </property>
   <property name="toolTip">
    <string>Список ночующих</string>
   </property>
  </action>
  <action name="act_command">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/img/edit_commands.png</normaloff>:/img/edit_commands.png</iconset>
   </property>
   <property name="text">
    <string>Команды</string>
   </property>
   <property name="toolTip">
    <string>Редактор команд</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
